docs/docs/APIdocs/layers/nn/Loss Functions/DiceCoefficientCriterion.md

```diff
@@ -0,0 +1,75 @@
+## DiceCoefficientCriterion ##
+
+**Scala:**
+```scala
+val loss = DiceCoefficientCriterion(sizeAverage=true, epsilon=1.0f)
+```
+**Python:**
+```python
+loss = DiceCoefficientCriterion(size_average,epsilon)
```

Comment:
This is a problem with DiceCoefficientCriterion - Scala has defaults while Python don't. Pls refer to "minor issues" at my comment. When the API changes the docs will be changed in sync. For now I just kept those difference in doc to make it sync with actual code.